sdk/inc/azure/core/az_context.h

```diff
@@ -27,58 +27,56 @@ typedef struct az_context az_context;
 /**
  * @brief A context is a node within a tree that represents expiration times and key/value
  * pairs.
- * @details The root node in the tree (ultimate parent) is #az_context_app which is a context for the
- * entire application. Each new node is a child of some parent.
+ * @details The root node in the tree (ultimate parent) is #az_context_app which is a context for
+ * the entire application. Each new node is a child of some parent.
  */
 struct az_context
 {
   struct
   {
     az_context const* parent; // Pointer to parent context (or NULL); immutable after creation
     int64_t expiration; // Time when context expires
-    void* key; // Pointers to the key & value (usually NULL)
-    void* value;
+    void const* key; // Pointers to the key & value (usually NULL)
+    void const* value;
   } _internal;
 };
 
 #define _az_CONTEXT_MAX_EXPIRATION 0x7FFFFFFFFFFFFFFF
 
 /**
- * @brief az_context_app is the ultimate root of all az_context instances. It allows you to cancel
- * your entire application. The az_context_app never expires but you can explicitly cancel it by
- * passing its address to az_context_cancel which effectively cancels all the az_context child
+ * @brief The ultimate root of all #az_context instances. It allows you to cancel
+ * your entire application. The #az_context_app never expires but you can explicitly cancel it by
+ * passing its address to #az_context_cancel which effectively cancels all the #az_context child
  * nodes.
  */
 extern az_context az_context_app;
 
 /**
- * @brief az_context_with_expiration creates a newexpiring az_context node that is a child of the
- * specified parent.
+ * @brief Creates a new expiring az_context node that is a child of the specified parent.
```

Comment:
Reads much better :)

Reply:
Thank you for bringing it up, I accidentally noticed that we missed `#` here :)